package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "bip-schnorr",
-  "version": "0.6.6",
+  "name": "@exodus/bip-schnorr",
+  "version": "0.6.6-fork-1",
```

Comment:
should be 
```suggestion
  "version": "0.6.6-exodus.1",
```

Reply:
Thanks @ChALkeR , I've already released `0.6.6-fork-1` to move forward with the integration. Could we use that label next time?